.github/update_starter_projects.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+import asyncio
+
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+src_path = os.path.join(project_root, "src")
+sys.path.insert(0, src_path)
+
+import backend.base.langflow.main
+from backend.base.langflow.initial_setup.setup import create_or_update_starter_projects
```

Comment:
Since Langflow will be installed when this is running, you should be better off importing the langflow module directly.

Reply:
Whoops, yes. Had this queued. Added